forever_style_guide.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "rails", "~> 4.2.1"
+  s.add_dependency "rails", "=4.2.0"
```

Comment:
@csto It looks like I have to use '=' for web app as well. For some reason, bundle install still install the latest version for me if I use "~> 4.2.0". Do you know how to work around it?